code/modules/reagents/chemistry/holder.dm

```diff
@@ -30,9 +30,11 @@
 		var/datum/chemical_reaction/D = new path()
 		var/list/reaction_ids = list()
 
-		if(D.required_reagents && D.required_reagents.len)
-			for(var/reaction in D.required_reagents)
-				reaction_ids += reaction
```

Comment:
Seemingly neither holder.dm or recipes.dm have any references to var/name or var/id???

Why do these vars exist?

Reply:
It is supposed to be used, but old coders did something incorrect.